src/interface/storage.thrift

```diff
@@ -310,6 +310,61 @@ struct UpdateEdgeRequest {
     7: bool insertable,
 }
 
+struct ScanEdgeRequest {
+    1: common.GraphSpaceID space_id,
+    2: common.PartitionID part_id,
+    // start key of this block
+    3: optional binary cursor,
+    // If specified, only return specified columns, if not, no columns are returned
+    4: map<common.EdgeType, list<PropDef>> (cpp.template = "std::unordered_map") return_columns,
```

Comment:
Why not use list directly?

Reply:
In go we need to collect properties of vertex/edge to generate a table with same columns. But in scan I suppose we only to collect some properties among different edge types, we don't care about what the schema looks like of other edge types, the rows in the table may have different rows. I think the map is better here.